### PR TITLE
fix: reviewing TODOes at v19

### PR DESCRIPTION
**Pull request by knst**
## Issue being fixed or feature implemented
During reviewing TODO were found some TODOes that can be done now.

## What was done?
 - fix: follow-up dash#3467 - replaced commented code to disabled code
 - follow-up bitcoin#16394 - uncommented code related to `watchonly` feature
 - removed out-dated TODO in `rpc/masternode.cpp` (already done)
 - fix: renamed name of clean up test_unittests: removed TODO and updated name of variable TRAVIS
 - rewritten todo inside `.travis.yml`
 - fix: adds a missing description for result of rpc `mnsync`

Last commit (`mnsync`) is an only candidate for backport to v19, other changes are non significant.

## How Has This Been Tested?
Run functional/unit tests

## Breaking Changes
No breaking changes

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

File: test/functional/feature_dbcrash.py [hunk 1]
```diff
@@ -242,14 +244,15 @@ def run_test(self):
...
-            #         self.nodes[3].invalidateblock(self.nodes[3].getblockhash(random_height))
+            skip_this_test_ReplayBlocks = True
+            random_height = random.randint(starting_tip_height, current_height)
+            self.log.debug("At height %d, considering height %d", current_height, random_height)
+            if not skip_this_test_ReplayBlocks and random_height > starting_tip_height:
+                # Randomly reorg from this point with some probability (1/4 for
+                # tip, 1/5 for tip-1, ...)
+                if random.random() < 1.0 / (current_height + 4 - random_height):
+                    self.log.debug("Invalidating block at height %d", random_height)
+                    self.nodes[3].invalidateblock(self.nodes[3].getblockhash(random_height))
```
Comment (PastaPastaPasta) [inline test/functional/feature_dbcrash.py:255]: What's the point of un-commenting if it's still disabled?

**Comment by PastaPastaPasta** [comment]
Seems fine except for [fix: follow-up dash#3467 - uncommenting code (but still disabled)](https://github.com/dashpay/dash/pull/5303/commits/78e8a57afb89b56176844c506f581d7a71789b7d)

File: test/functional/feature_dbcrash.py (hunk 1 above)
Comment (UdjinM6) [reply test/functional/feature_dbcrash.py:255]: less merge conflicts I guess

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

File: test/functional/feature_dbcrash.py (hunk 1 above)
Comment (knst) [reply test/functional/feature_dbcrash.py:255]: For make code "compilable". Comments arr jst ignored

**Pull request merged**